modules/apps/29-fee/keeper/events.go

```diff
@@ -37,10 +37,6 @@ func EmitIncentivizedPacketEvent(ctx sdk.Context, packetID channeltypes.PacketId
 			sdk.NewAttribute(types.AttributeKeyAckFee, totalAckFees.String()),
 			sdk.NewAttribute(types.AttributeKeyTimeoutFee, totalTimeoutFees.String()),
 		),
-		sdk.NewEvent(
-			sdk.EventTypeMessage,
-			sdk.NewAttribute(sdk.AttributeKeyModule, types.ModuleName),
```

Comment:
I've been waiting to do this for so long haha

Reply:
😂

Reply:
Sadly, I'm having to re-add these events! 🥲 https://github.com/cosmos/cosmos-sdk/issues/14295